tests/Resources/keywords_avdelning_mottagning.robot

```diff
@@ -0,0 +1,59 @@
+*** Settings ***
+Documentation
+Library  SeleniumLibrary
+
+*** Keywords ***
+
+Expandera Kontaktblock Test
+
+    ${width}	${height} =	            Get Element Size	css:#main-content > div > div.department-wrapper > div.main-wrapper.department-one > div.row > div > div > div.kivcontact__column-container
+    Log                                 ${height}
+    Sleep                               1s
+    Click Element                       css:#main-content > div > div.department-wrapper > div.main-wrapper.department-one > div.row > div > div > div.block__row.kivcontact__expandrow > div
+    Sleep                               1s
+    ${width2}	${height2} =	        Get Element Size	css:#main-content > div > div.department-wrapper > div.main-wrapper.department-one > div.row > div
+    Log                                 ${height2}
+    Should Be True                      ${height} < ${height2}
+
+Test Flikar
+
+    Wait Until Element Contains        css:#tab-1 > div > div > div > div > div > div.block__header    VÅRD OCH HÄLSA
+    Sleep                               1s
+    Click Element                       css:#main-content > div > div.department-wrapper > div.main-wrapper.department-three > div:nth-child(7) > div > div.vgr-tabs__bar > ul > li:nth-child(2)
+    Wait Until Element Contains         css:#tab-2 > div > div > div > div > div > div.block__header    BESÖK HOS OSS
+    Sleep                               1s
+    Click Element                       css:#main-content > div > div.department-wrapper > div.main-wrapper.department-three > div:nth-child(7) > div > div.vgr-tabs__bar > ul > li:nth-child(3)
+    Wait Until Element Contains         css:#tab-3 > div > div > div > div > div > div.block__header > h2   VÄLJ VÅR VÅRDCENTRAL
+
+Test Kontaktuppgifter Block Info Is Correct
+
+    Click Element                       css:#main-content > div > div.department-wrapper > div.main-wrapper.department-one > div.row > div > div > div.block__row.kivcontact__expandrow > div
+    Wait Until Element Contains         css:#main-content > div > div.department-wrapper > div.main-wrapper.department-one > div.row > div > div > div.kivcontact__column-container > div:nth-child(1) > div:nth-child(1) > div.media-body    031
```

Comment:
again, many very complex selectors. It should be possible to streamline them a little. How do we anticipate them to fail? I guess it's most likely they wont show data at all, rather than mix up the order of it. so maybe `.kivcontact__column-container` is specific enough (if there's only one "kiv-block" on the page)